ostree-backup.sh

```diff
@@ -63,7 +63,7 @@ else
 fi
 
 if [[ ! -f /var/tmp/backup/etc.tgz ]]; then
-    ostree admin config-diff | awk '{print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
+    ostree admin config-diff | awk '$1 != "D" {print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
```

Comment:
We also need to record the deletions, so we can delete them as part of the upgrade prep

Reply:
Ok, I see. For testing purposes in my env, I have disabled chronyd and tried the command, but it fails with the below error, in case's useful.

```
-> systemctl disable chronyd.service 
Removed "/etc/systemd/system/multi-user.target.wants/chronyd.service".

-> ostree admin config-diff | awk '{print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
tar: Removing leading `/' from member names
tar: Removing leading `/' from hard link targets
tar: /etc/systemd/system/multi-user.target.wants/chronyd.service: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
```